libkpod/remove.go

```diff
@@ -0,0 +1,46 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+)
+
+// Remove removes a r
+func (c *ContainerServer) Remove(container string, force bool) (string, error) {
```

Comment:
Can we change Server to call into these, and remove the duplicate implementations there? The semantics will be slightly different, but shouldn't be that hard to match.